etc/kayobe/ansible/upload-image.yml

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
upload-image.yml might be a bit generic. pulp-upload-image.yml?

Reply:
Done